src/Pact/Analyze/Parse/Prop.hs

```diff
@@ -118,6 +124,13 @@ expToPreProp = \case
 
   ParenList [EAtom' SObjectProjection, ELiteral' (LString objIx), obj]
     -> PreAt objIx <$> expToPreProp obj
+  ParenList [EAtom' SPropRead, tn, rk, ba] -> PrePropRead
+    <$> expToPreProp tn
+    <*> expToPreProp rk
+    <*> expToPreProp ba
+  exp@(ParenList [EAtom' SPropRead, _tn, _rk]) -> throwErrorIn exp
+    "read must specify a time ('before or 'after). example: (= result (read accounts user 'before))"
```

Comment:
We should probably use the symbol (twice) instead of hard-coding `read` here, in case we change its name